### PR TITLE
Open external links in new tabs on public pages

### LINKED ISSUE #1823: FAQ's: Please make all links open in a new tab
RC
SCM Program
Links to URL's should open in a new tab. 
(likely an issue in other program's as well -- so it would make sense to change this across this board)

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Fix  #1823

For testing, try adding some links in program page content through cms, and make sure those links open in a new tab.

### CODE REVIEW

**Comment by noisecapella** [comment]
Are we intending this only to be for external links?

File: static/js/entry/public.js [hunk 1]
```diff
@@ -179,4 +179,6 @@ $(document).ready(function ($) {
     $(".accordion-content").not($(this).next()).slideUp('fast').prev().
     find('.material-icons').removeClass('rotate-reset').addClass('rotate');
   });
+  // All external links should open in new tab
+  $('a[href^="https://"]').attr('target', '_blank');
```
Comment (noisecapella) [inline static/js/entry/public.js]: This only works for `https` links, can you update for `http` as well?

**Comment by annagav** [comment]
I understood the issue as concerning only external links. @pdpinch what do you think?

**Comment by noisecapella** [comment]
Looks good to me, feel free to merge when @pdpinch confirms that we only want to do this for external links :+1:

**Comment by Ferdi** [comment]
Required reading before this is merged  :-)

https://www.smashingmagazine.com/2008/07/should-links-open-in-new-windows/  

/cc @roberthouse54

**Comment by roberthouse54** [comment]
Might as well also read this, for the counter opinion: http://uxmovement.com/navigation/why-external-links-should-open-in-new-tabs/

**Comment by Ferdi** [comment]
Yes, please also read the comments on the article you linked and come up with some recommendations about what to do in our case

**Comment by roberthouse54** [comment]
I honestly do not have much of an opinion on this. I think is is one of those things that is debated as a UX thing, but I do not feel strongly about it one way or the other. Jared opened this issue for some reason, but I am not sure why. I think we should just do it, but I also do not think it is a very big issue.

**Pull request merged**